saul-examples/src/main/scala/edu/illinois/cs/cogcomp/saulexamples/mSpRL2017/README.md

```diff
@@ -163,489 +163,230 @@ Note: The training or test relation created in this example are not playing any
+public-sign            100.000  2.239  4.380    134      3
+river                    9.091  0.621  1.163    161     11
+road                     0.000  0.000  0.000     63      0
+rock                    27.778  2.366  4.360    634     54
+roof                    25.000  2.222  4.082     90      8
+ruin-archeological       4.671 32.353  8.163     68    471
+sand-beach               8.225 12.102  9.794    157    231
+sand-dessert            10.164 30.392 15.233    102    305
+seal                     3.253 48.718  6.100     39    584
+ship                     4.762  2.326  3.125     43     21
+sidewalk                15.152  5.556  8.130    180     66
+sky                     37.864 56.522 45.349    552    824
+sky-blue                68.773 65.141 66.908   1136   1076
+sky-light               61.323 65.135 63.172    370    393
+sky-night               30.435 81.667 44.344     60    161
+sky-red-sunset-dusk     26.271 51.667 34.831     60    118
+snow                     7.639 21.782 11.311    101    288
+stairs                   0.000  0.000  0.000     57      6
+statue                   0.000  0.000  0.000     49      2
+street                  19.388  9.268 12.541    205     98
+table                    0.000  0.000  0.000    105      4
+tower                    3.448  2.857  3.125     35     29
+tree                    13.539 21.673 16.667    263    421
+trees                   23.952 22.099 22.989    543    501
+trunk                   18.182 18.367 18.274     98     99
+umbrella                 8.333  3.125  4.545     32     12
+vegetation              33.153 32.624 32.887    564    555
+wall                    41.509  4.331  7.843    508     53
+water                   33.333  0.719  1.408    139      3
+water-reflection        17.500 11.667 14.000     60     40
+waterfall                6.759 67.241 12.283     58    577
+window                  18.182  6.006  9.029    333    110
+woman                   21.212  1.381  2.593    507     33
+wood                     0.000  0.000  0.000     85     16
+wooden-furniture         7.491 17.699 10.526    113    267
+        ----------------------------------------------------------
+Accuracy                23.317   -      -      -     18317
+
 
 </pre>
 ## Running
-To run the main app with default properties:
+Please download the full dataset from ImageCLEF (http://www.imageclef.org/SIAPRdata).  
+Make sure you download all the three parts saiaprtc12ok.part1.rar, saiaprtc12ok.part2.rar and saiaprtc12ok.part3.rar.
+Once you download, please extract saiaprtc12ok.part1.rar, it will automatically extract the other two parts and make folders namely matlab and benchmark.
+Inside benchmark folder you will find saiapr_tc-12 folder, move the saiapr_tc-12 folder to saul -> data/mSprl folder.
+From matlab folder move training.mat, testing.mat and wlist100.txt files inside saul -> data/mSprl/saiapr_tc-12 folder.
+
+After completing the above task, run the main app with default properties:
 
```

Comment:
This is good to me. Only the name conventions should be checked, otherwise good to merge.